spec/controllers/archive_faqs_controller_spec.rb

```diff
@@ -135,5 +234,155 @@
+    end
+
+    context "for a non-default locale" do
+      let(:locale) { non_standard_locale.iso }
+      it_behaves_like "an action translation authorized admins can access"
+    end
+  end
+
+  describe "GET #new" do
+    subject { get :new, params: { language_id: locale } }
+    let(:success) do
+      expect(response).to render_template(:new)
+    end
+
+    context "for the default locale" do
+      let(:locale) { "en" }
+      it_behaves_like "an action only fully authorized admins can access"
+    end
+
+    context "for a non-default locale" do
+      let(:locale) { non_standard_locale.iso }
+      it_behaves_like "a non-English action that nobody can access"
+    end
+  end
+
+  describe "POST #create" do
+    subject { post :create, params: { archive_faq: attributes_for(:archive_faq), language_id: locale } }
+    let(:success) do
+      expect(ArchiveFaq.count).to eq(1)
+      it_redirects_to_with_notice(assigns[:archive_faq], "ArchiveFaq was successfully created.")
+    end
+
+    context "for the default locale" do
+      let(:locale) { I18n.default_locale }
+      it_behaves_like "an action only fully authorized admins can access"
+    end
+
+    context "for a non-default locale" do
+      let(:locale) { non_standard_locale.iso }
+      it_behaves_like "a non-English action that nobody can access"
+    end
+  end
+
+  describe "GET #manage" do
+    subject { get :manage, params: { language_id: locale } }
+    let(:success) do
+      expect(response).to render_template(:manage)
+    end
+
+    context "for the default locale" do
+      let(:locale) { "en" }
+      it_behaves_like "an action only fully authorized admins can access"
+    end
+
+    context "for a non-default locale" do
+      let(:locale) { non_standard_locale.iso }
+      it_behaves_like "a non-English action that nobody can access"
+    end
+  end
+
+  describe "POST #update_positions" do
+    let!(:faq1) { create(:archive_faq, position: 1) }
+    let!(:faq2) { create(:archive_faq, position: 2) }
+    let!(:faq3) { create(:archive_faq, position: 3) }
+    subject { post :update_positions, params: { archive_faqs: [3, 1, 2], language_id: locale } }
+    let(:success) do
+      expect(faq1.reload.position).to eq(3)
+      expect(faq2.reload.position).to eq(1)
+      expect(faq3.reload.position).to eq(2)
+      it_redirects_to_with_notice(archive_faqs_path, "Archive FAQs order was successfully updated.")
+    end
+
+    context "for the default locale" do
+      let(:locale) { I18n.default_locale }
+      it_behaves_like "an action only fully authorized admins can access"
+    end
+
+    context "for a non-default locale" do
+      let(:locale) { non_standard_locale.iso }
+      it_behaves_like "a non-English action that nobody can access"
+    end
+  end
+
+  describe "GET #confirm_delete" do
+    let(:faq) { create(:archive_faq) }
+    subject { get :confirm_delete, params: { id: faq, language_id: locale } }
```

Comment:
I don't think we have the cop for it turned on, but the recommended order for subject/let is `subject` first, followed by `let` or `let!`: https://github.com/rubocop/rspec-style-guide?tab=readme-ov-file#declaring-subject-letlet-and-beforeafter-hooks. I'm not particularly attached to that order, but it would be better IMO to have the `let`s together before or after the `subject`